internal/wasmldr/src/loader/configured/mod.rs

```diff
@@ -53,21 +52,39 @@ impl Loader<Configured> {
     }
 
     pub fn next(self) -> Result<Loader<Requested>> {
-        // TODO:
-        //   1. get attestation
-        //   2. get intermediate cert
-        //   3. choose key type based on technology
-        //   4. generate tech-specific attestation extension request
+        // TODO: get intermediate cert
+
+        let platform = Platform::get()?;
+        let cert_algo = match platform.technology() {
+            Technology::Snp => SECP_384_R_1,
+            _ => SECP_256_R_1,
```

Comment:
Let's list each technology here rather than using a fallback. The reason is this so that this code breaks when we add a new technology rather than silently doing the wrong thing.